src/componentVersions.js

```diff
@@ -1,3 +1,11 @@
+// To use this in other pages, add the following to the top of your page
+// ```mdx-code-block
+// import {versions} from '@site/src/componentVersions';
+// import CodeBlock from '@theme/CodeBlock';
+// ```
+// Then use <>{versions.yourThing}</> to add the version in place.
+// Note that this does not render correctly in markdown ables so you need to use additional steps, see docs/modeling-your-data/modeling-your-data-with-dbt/index.md as an example
```

Comment:
There is some info about this in the README also. Can we improve the README (I think it has a missing link) and redirect the reader there instead?